Sources/SwiftInspectorVisitors/Tests/TypeSyntaxVisitorSpec.swift

```diff
@@ -95,12 +95,9 @@ final class TypeSyntaxVisitorSpec: QuickSpec {
       }
 
       context("when there are no properties") {
-        beforeEach {
```

Comment:
With `VisitorExecutor` we don't need these `beforeEach(...)` invocations.

Reply:
`let content`s are promoted to the parent scope with no changes.

Reply:
Tip to Francisco if he reviews after me: review this commit with whitespace changes ignored.